### PR TITLE
Feature image recognition

### PR DESCRIPTION
**Pull request by AleksandrasMaliuginas**
EmguCV loading check added.

### CODE REVIEW

**Review by imila254 — APPROVED** [review]
Looks good.

**Review by sk057 — APPROVED** [review]

File: Controllers/ImageRecognizer.cs [hunk 1]
```diff
@@ -33,29 +50,62 @@ public void RecognizeTestImages()
...
             }
         }
 
-        private string Recognize(FileInfo imagePath)
+        private string Recognize(FileSystemInfo imagePath)
         {
             Debug.WriteLine(imagePath);
 
-            // Original Image
+            #region Original Image
```
Comment (barkas90) [inline Controllers/ImageRecognizer.cs:60]: What is #region ?

File: Controllers/ImageRecognizer.cs [hunk 2]
```diff
@@ -31,29 +50,62 @@ public void RecognizeTestImages()
             catch (Exception e)
             {
                 Debug.WriteLine(e);
-                throw;
             }
         }
 
-        private string Recognize(FileInfo imagePath)
+        private string Recognize(FileSystemInfo imagePath)
```
Comment (sk057) [inline Controllers/ImageRecognizer.cs:56]: Why change this?

File: Controllers/ImageRecognizer.cs (hunk 1 above)
Comment (AleksandrasMaliuginas) [reply Controllers/ImageRecognizer.cs:60]: [#region](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region)

File: Controllers/ImageRecognizer.cs (hunk 2 above)
Comment (AleksandrasMaliuginas) [reply Controllers/ImageRecognizer.cs:56]: Resharper suggested :D As I understand, it makes parameter more general.

**Pull request merged**